web/src/components/FormAndReviewList.js

```diff
@@ -8,20 +8,27 @@ const FormAndReviewItem = ({
   initialCollapsed,
   ...rest
 }) => {
+  const container = useRef(null);
   const [collapsed, setCollapsed] = useState(initialCollapsed);
-  const collapse = useCallback(() => setCollapsed(true), []);
+  const collapse = useCallback(() => {
+    const { top } = container.current.getBoundingClientRect();
+    if (top < 0 || top > window.innerHeight) {
```

Comment:
This is just to make sure the target isn't already in view. The browser doesn't check for us, and will scroll the element to the top of the viewport. I thought we probably didn't want to scroll the page to the top if the target was already in view - that'd be especially jarring.